starter/cms-payload/app/head.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Head() {
```

Comment:
Could you migrate this to the Metadata API? 

https://nextjs.org/docs/app/building-your-application/optimizing/metadata